### PR TITLE
MR-25 Row with Background

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# To Do:
- [x] CSS to SCSS

# Summary
- Move Row with background styling from Math-Prod site CSS Injector and into SASS.

# Needed By (Date)
- N/A

# Urgency
- Medium

# Steps to Test
1. Checkout this branch
2. Clear cache

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by rebeccahongsf** [comment]
I'll be taking a look and resolving the Code climate issues today!

File: docroot/themes/humsci/su_humsci_theme/patterns/molecules/hero-text-overlay/hero-text-overlay.html.twig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% set attributes = attributes.addClass('hero-text-overlay') %}
```
Comment (pookmish) [inline docroot/themes/humsci/su_humsci_theme/patterns/molecules/hero-text-overlay/hero-text-overlay.html.twig]: I dont think this file relates to this PR.

File: docroot/themes/humsci/su_humsci_theme/patterns/molecules/hero-text-overlay/hero-text-overlay.ui_patterns.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+hero_text_overlay:
```
Comment (pookmish) [inline docroot/themes/humsci/su_humsci_theme/patterns/molecules/hero-text-overlay/hero-text-overlay.ui_patterns.yml]: I dont think this file relates to this PR.

**Review by pookmish — CHANGES_REQUESTED** [review]
A couple of files that arent related to the PR here

**Comment by rebeccahongsf** [comment]
Removed irrelevant files! :)

File: docroot/themes/humsci/mathematics/scss/ckeditor.scss [hunk 3]
```diff
@@ -1,3 +1,5 @@
 @charset "UTF-8";
 
-@import 'base';
+@import
+'base',
+'row-with-background';
```
Comment (pookmish) [inline docroot/themes/humsci/mathematics/scss/ckeditor.scss]: ckeditor doesnt need this stylesheet included

File: docroot/themes/humsci/mathematics/scss/row-with-background.scss [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+//
+// Mathematics
+// Mathematics website specific styles.
+//
+// Decanter
+@import 'decanter-no-markup';
+// Decanter Overrides
+@import '../../su_humsci_theme/scss/decanter-overrides/decanter-overrides';
+// SU HumSci
+@import '../../su_humsci_theme/scss/utilities/mixins/index';
```
Comment (pookmish) [inline docroot/themes/humsci/mathematics/scss/row-with-background.scss]: These imports aren't being used in any capacity. we dont need them

File: docroot/themes/humsci/mathematics/scss/row-with-background.scss [hunk 5]
```diff
@@ -0,0 +1,261 @@
+@charset "UTF-8";
+
+//
+// Mathematics
+// Mathematics website specific styles.
```
Comment (pookmish) [inline docroot/themes/humsci/mathematics/scss/row-with-background.scss]: Update comment

**Review by pookmish — CHANGES_REQUESTED** [review]
Just some unnecessary imports

**Pull request merged**